src/lib/Scenes/Collection/Components/FeaturedArtists.tsx

```diff
@@ -8,24 +8,17 @@ import React from "react"
 import { TouchableHighlight } from "react-native"
 import { createFragmentContainer, graphql } from "react-relay"
 import { TrackingProp } from "react-tracking"
+import styled from "styled-components"
 
 interface FeaturedArtistsProps {
   collection: FeaturedArtists_collection
   tracking?: TrackingProp
 }
 
-interface FeaturedArtistsState {
-  showMore: boolean
-}
-
-const track: Track<FeaturedArtistsProps, FeaturedArtistsState> = _track
+const track: Track<FeaturedArtistsProps, any> = _track
```

Comment:
Since there is no state associated with this component anymore, I think it's more accurate for the state type to be `{}` instead of `any`.

I don't think it really matters a whole lot to any consumer of this component, but it's best to avoid `any` when we can. 

```suggestion
const track: Track<FeaturedArtistsProps, {}> = _track
```